scipy/special/__init__.py

```diff
@@ -280,7 +280,7 @@
    :toctree: generated/
 
    gamma        -- Gamma function.
-   gammaln      -- Log of the absolute value of the gamma function.
+   gammaln      -- Log transformation of the gamma function.
```

Comment:
Did you use the word "transformation" to point out that it is not always log of the gamma function? Looks like a good solution.